nabs/plans.py

```diff
@@ -501,13 +503,14 @@ def daq_ascan(detectors, motor, start, end, nsteps):
     :py:func:`nabs.preprocessors.daq_step_scan_decorator`.
     """
 
-    yield from bp.scan(detectors, motor, start, end, nsteps)
+    yield from bp.scan(detectors, motor, start, end, n)
 
 
 @bpp.reset_positions_decorator()
 @bpp.relative_set_decorator()
 @nbpp.daq_step_scan_decorator
-def daq_dscan(detectors, motor, start, end, nsteps):
+@nbpp.step_size_decorator
```

Comment:
I like that you boiled this down to a decorator :+1:

Reply:
💯 % - I would not have thought this could be done so cleanly.